### ISSUE TITLE
Bug on Rolento's Stage

### ISSUE THREAD

**Issue by GorosLair** [originalgrego/FinalFightAE-Installer#8]
Hey Grego, how's things?

I do not know whether this message will reach you, but was this bug fixed?

https://ibb.co/rmfLGh8

I wanna know whether you have a version with this fixed... but not the CPS2 version (because it's quite different from the original game, and I want the original one with 3 players only and the bugs fixed), because I'd like to have CPS1 version with only this fixed, as well the Haggar Dynamite too.

I know there's the ffightae v2 cps1 version... but were those bugs fixed? If it's possible, could send a fixed version like I've written above. Thanks in advance and, on time, I wish all the best for you.

**Comment by thomasgoldstein** [comment]
I could be wrong, but if it's the same as with issue #3, then it only got fixed in the CPS-2 version. I'm under the impression the CPS-1 version has been abandoned for the CPS-2 one.

**Comment by GorosLair** [comment]
> I could be wrong, but if it's the same as with issue #3, then it only got fixed in the CPS-2 version. I'm under the impression the CPS-1 version has been abandoned for the CPS-2 one.

No, It's not the same bug, buddy. It's sad to know that once what I really wanted was to get the Original Game with 3 players only... but the new version is very good, but it doesn't keep the original aspect. Because it's got new enemies, new colors, different names, etc. Under these conditions, I prefer to keep the old one, even with the bug. Anyway, thanks for letting me know and I hope Grego might be able to help me anyway.

**Comment by thomasgoldstein** [comment]
I was not saying it was the same bug. I was giving an example of a bug that was only fixed in the CPS-2 version.
